tests/improver-blend-adjacent-points/02_basic_mean.bats

```diff
@@ -44,7 +44,10 @@
 
   # Run nccmp to compare the output and kgo.
   improver_compare_output "$TEST_DIR/output.nc" \
-      "$IMPROVER_ACC_TEST_DIR/blend_adjacent_points/basic_mean/kgo.nc"
-  rm "$TEST_DIR/output.nc"
-  rmdir "$TEST_DIR"
+      "$IMPROVER_ACC_TEST_DIR/$KGO"
+
+  if [ -n "$RECREATE_BATS_KGO" ]; then
```

Comment:
[`[[` preferable to `[` ](https://google.github.io/styleguide/shell.xml?showone=Test,_%5B_and_%5B%5B#Test,_%5B_and_%5B%5B) (like an upgrade!)
I guess this is always set?

Reply:
Changed to double brackets.
I hope it normally not set. If it's not set, then the tests run normally, if it is set then data will be created in the path it points to, e.g. ```export RECREATE_BATS_KGO=/path/to/new/tests```